src/Pdo/PdoUpdateCapableTrait.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace RebelCode\Storage\Resource\Pdo;
+
+use Dhii\Expression\ExpressionInterface;
+use Dhii\Expression\LogicalExpressionInterface;
+use Dhii\Util\String\StringableInterface as Stringable;
+use Exception as RootException;
+use InvalidArgumentException;
+use PDOStatement;
+
+/**
+ * Common functionality for objects that can update records in a database using PDO.
+ *
+ * @since [*next-version*]
+ */
+trait PdoUpdateCapableTrait
+{
+    /**
+     * Executes an UPDATE SQL query, updating records in the database that satisfy the given condition.
+     *
+     * @since [*next-version*]
+     *
+     * @param array                           $changeSet A map of field names mapping to the values to change.
```

Comment:
Can these be SQL expressions? What happens if they are? What is this map expected to contain?

Reply:
> Can these be SQL expressions?

I guess they can be. I considered this for a while, but I hadn't found whether expressions are supported in UPDATE queries or whether it was right to support them even if only a few SQL databases supported them.

> What happens if they are?

This file would have more characters.

> What is this map expected to contain?

Field names mapping to new values. These are used to build the `SET` portion of the query.

```
UPDATE my_table SET fieldName = "value", otherField = "other value";
```